### DISCUSSION TITLE
zh/guide/giscus

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#458] [Announcements]
# zh/guide/giscus

Giscus 是一个基于 GitHub Discussion 的评论系统，启用简便。 准备工作 1. 你需要创建一个公开仓库，并开启评论区，以作为评论存放的地点 1. 你需要安装 Giscus App (https://github.com/apps/giscus)，使其有权限访问对应仓库。 1. 在完成以上步骤后，请前往 Giscus 页面 (htt...

https://plugin-comment2.vuejs.press/zh/guide/giscus.html

**Comment by victorting** [comment]
测试
